src/Handler/Build.php

```diff
@@ -44,7 +44,7 @@ public function __construct($cwd)
         $vagrantTasks = [
             'up --no-provision',
             'provision',
-            'ssh -- "cd /srv/app && make install"',
+            'ssh -- "cd /srv/app && composer install --verbose && bin/console doctrine:database:create --if-not-exists"',
```

Comment:
You're right, maybe calling the makefile stay the simpler way... as it is created per env.
If you have some suggestion... I take :)